app/src/main/java/com/asfoundation/wallet/ui/iab/IabActivity.java

```diff
@@ -179,12 +180,13 @@ public static Intent newIntent(Activity activity, String url) {
   @Override
   public void showLocalPayment(String domain, String skuId, String originalAmount, String currency,
       String bonus, String selectedPaymentMethod, String developerAddress, String type,
-      BigDecimal amount, String callbackUrl, String orderReference, String payload) {
+      BigDecimal amount, String callbackUrl, String orderReference, String payload,
+      boolean validBonus) {
```

Comment:
We already have a bonus value. Aren't we able to use that value to control if the animation to is supposed to be shown or not

Reply:
The bonus value contains the currency sign. That's why I decided to use the same approach as the topup

Reply:
@JoaoRaimundo you're correct and the solution could be improved